src/server/views/dashboard/lists.njk

```diff
@@ -14,7 +14,22 @@
         </div>
       {% endif %}
     </div>
-    <table class="govuk-table dashboard-table">
+
+    <div class="govuk-grid-row govuk-!-margin-top-2">
+      {% for dashboardBox in dashboardBoxes %}
+        <div class="govuk-grid-column-one-third govuk-!-padding-top-3 dashboard-box--{{dashboardBox.cssClass}}">
+          <strong class="govuk-body govuk-!-font-weight-bold">{{ _.startCase(dashboardBox.name)}}</strong>
+          {% if dashboardBox.cssClass === "red" %}
```

Comment:
similar to my comment above.. why does the colour red determine why something is sorted by asc or not. It makes the code less explicit, you are making a developer "guess" red = bad. but in future, the colour might be yellow, so this code breaks. 

I think `warning` or `error` works.